### PR TITLE
[MDS-5332] Fixing issue in updating mine status

### PR DESCRIPTION
**Pull request by isuru-aot**
## Objective 

[MDS-5332](https://bcmines.atlassian.net/browse/MDS-5332)

_Why are you making this change? Provide a short explanation and/or screenshots_

-  Fix the mine status selection
- Validate the status being updated correctly

Screenshot 2023-06-13 170057

### CODE REVIEW

**Review by henryoforeh-dev — APPROVED** [review]

**Review by matbusby-fw — APPROVED** [review]
LGTM!

File: services/core-web/src/components/common/RenderCascader.js [hunk 1]
```diff
@@ -23,28 +23,43 @@ const defaultProps = {
...
+    >
+      <Cascader
+        expandTrigger="hover"
+        id={props.id}
+        placeholder={props.placeholder}
+        options={props.options}
+        {...props.input}
+        changeOnSelect={props.changeOnSelect}
+        defaultValue={selectedValue}
+        value={handleCascaderChange(selectedValue)}
```
Comment (taraepp) [inline services/core-web/src/components/common/RenderCascader.js:57]: I am curious why the change function is here for the value!

**Review by asinn134 — APPROVED** [review]

File: services/core-web/src/components/common/RenderCascader.js (hunk 1 above)
Comment (isuru-aot) [reply services/core-web/src/components/common/RenderCascader.js:57]: It handles the updates to the selected value. Without that the updated value doesn't get updated correctly.

Comment (taraepp) [reply services/core-web/src/components/common/RenderCascader.js:57]: Interesting! I am surprised that it works for this attribute because it doesn't return a value. I have a feeling that if it were to be refactored to typescript that it would give an error.
